src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -0,0 +1,108 @@
+import React, {
+  MouseEventHandler,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+import classNames from "classnames";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBookmark as solidBookmark } from "@fortawesome/free-solid-svg-icons";
+import { faBookmark as regularBookmark } from "@fortawesome/free-regular-svg-icons";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { WithVenueId } from "utils/id";
+import { isTruthy } from "utils/types";
+import { isEventLive } from "utils/event";
+
+import { ONE_HOUR_IN_MINUTES } from "utils/time";
+
+import { saveEventToProfile } from "api/profile";
+import { useUser } from "hooks/useUser";
+
+import { calcStartPosition, HOUR_WIDTH } from "../Schedule/Schedule.utils";
+
+import "./ScheduleEvent.scss";
+
+export interface ScheduleEventProps {
+  event: WithVenueId<PersonalizedVenueEvent>;
+  scheduleStartHour: number;
+  isUsers?: boolean;
+}
+
+export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
+  event,
+  scheduleStartHour,
+  isUsers,
+}) => {
+  const [isBookmarked, setBookmark] = useState(event.isSaved);
+
+  useEffect(() => {
+    setBookmark(event.isSaved);
+  }, [event.isSaved]);
+
+  const { userId } = useUser();
+
+  const containerClasses = useMemo(
+    () =>
+      classNames(
+        "ScheduleEvent",
+        {
+          "ScheduleEvent--live": isEventLive(event),
+        },
+        { "ScheduleEvent--users": isTruthy(isUsers) }
+      ),
+    [event, isUsers]
+  );
```

Comment:
ohh, got that. I hope to keep in mind that next time so that you wouldn't need to remind me every time. :see_no_evil: 
fixed: https://github.com/sparkletown/sparkle/commit/28b4cc25e369c09ac8ce1a6b2e3f4ee709068144

Reply:
Haha, all good :) Pretty sure you had already written both cases of it before I did this round of review.

Reply:
Though looking at https://github.com/sparkletown/sparkle/commit/28b4cc25e369c09ac8ce1a6b2e3f4ee709068144 you're still passing 2 objects into `classNames` in `src/components/molecules/ScheduleEvent/ScheduleEvent.tsx` rather than just having them as 2 keys on the one object as per my example above in https://github.com/sparkletown/sparkle/pull/1302#discussion_r628997692

Reply:
ooops, done here: https://github.com/sparkletown/sparkle/pull/1302/commits/84c633a2848faab78da90654662b3d1b8aba1942